ui/ducks/metamask/metamask.js

```diff
@@ -279,15 +284,13 @@ export function getBlockGasLimit(state) {
   return state.metamask.currentBlockGasLimit;
 }
 
-export function getConversionRate(state) {
-  return state.metamask.conversionRate;
+export function getNativeCurrency(state) {
+  return getProviderConfig(state).ticker;
 }
 
-export function getNativeCurrency(state) {
-  const useCurrencyRateCheck = getUseCurrencyRateCheck(state);
```

Comment:
I don't understand why this ternary was here before... Why would this impact which of these two sources of data would be used? Wouldn't/shouldn't they have been the same?

Reply:
It's possible they aren't the same in the case that the current network has just changed. This is because `pendingNativeCurrency` is set as an intermediary placeholder, meaning any currency rate conversion in state still reflects the previous native currency. On the next CurrencyRateController poll loop `pendingNativeCurrency` is set to `nativeCurrency` and the currency rates are updated to reflect that.

The new CurrencyRateController changes get rid of this pattern, so this ternary is unnecessary now